shift_detector/checks/simple_check.py

```diff
@@ -22,13 +22,13 @@ def __init__(self, categorical_threshold=0.05, mean_threshold=0.15, median_thres
                                   'num_distinct': num_distinct_threshold, 'std': std_threshold, 'completeness':
                                   completeness_threshold}
 
-        if categorical_threshold < 0 or categorical_threshold > 1:
-            raise ValueError('The categorical threshold of {} is not correct. It has be between the values of '
+        if categorical_threshold < 0:
+            raise ValueError('The categorical threshold of {} is not correct. It has to be between the values of '
                              '0 and 1.'.format(categorical_threshold))
 
         for t_name, t_value in threshold_names_values.items():
-            if t_value < 0 or t_value > 1:
-                raise ValueError('The {}_threshold of {} is not correct. It has be between the values of 0 and 1'
+            if t_value < 0:
+                raise ValueError('The {}_threshold of {} is not correct. It has to be between the values of 0 and 1'
```

Comment:
You are just testing for one direction now. This is not consistent with the error message...

Reply:
see comment above